client/docs/swagger-ui/swagger.yaml

```diff
@@ -31934,14 +31928,20 @@ definitions:
       added_at:
         type: string
         format: date-time
-    description: ExecutorWasm represents a wasm used for some execution in the protocol.
+      expiration_height:
+        type: string
+        format: int64
+        description: |-
+          ExpirationHeight represents the block height at which the oracle
+          program will be pruned. The value of zero means no expiration.
+    description: OracleProgram is a wasm used for data request.
   sedachain.wasm_storage.v1.QueryCoreContractRegistryResponse:
     type: object
     properties:
       address:
         type: string
     description: The response message for QueryCoreContractRegistry RPC.
-  sedachain.wasm_storage.v1.QueryDataRequestWasmResponse:
+  sedachain.wasm_storage.v1.QueryExecutorWasmResponse:
     type: object
```

Comment:
Maybe QueryDataRequestResponse would be better? Or is this the overlay Executor WASM again?

Reply:
In reality `QueryDataRequestWasmResponse` was renamed to `QueryOraclePrograms`.